_docs/durability.md

```diff
@@ -4,4 +4,138 @@ title: Durability
+
+### Serialize the rulebase separately from the working memory ###
+
+{% highlight clojure %}
+
+;;;; Define some files to demonstrate the different way to serialize session data.
+
+(def rulebase-file
+  "A file to hold the serialized session rulebase without the working memory."
+  (io/file "rulebase-file.dat"))
+
+(def working-memory-file
+  "A file to hold the serialized session working memory data without the rulebase."
+  (io/file "working-memory-file.dat"))
+
+;;;; Serializing only the rulebase of a session (not the working memory).
+
+(with-open [os (io/output-stream rulebase-file)]
+  (d/serialize-rulebase fired-session
+                        (df/create-session-serializer os)))
+
+(def deserialized-rulebase
+  (with-open [is (io/input-stream rulebase-file)]
+    (d/deserialize-rulebase (df/create-session-serializer is))))
+
+;;;; Serializing only the working memory of the session (not the rulebase).
+
+(def wm-serializer (->LocalMemorySerializer (atom nil)))
+
+(with-open [os (io/output-stream working-memory-file)]
+  (d/serialize-session-state fired-session
+                             (df/create-session-serializer os)
+                             wm-serializer))
+(def deserialized-session
+  (with-open [is (io/input-stream working-memory-file)]
+    (d/deserialize-session-state (df/create-session-serializer is)
+                                 wm-serializer
+                                 ;; NOTE!! The deserialization needs a rulebase to attach the working
+                                 ;; memory to to fully restore the session.  They were stored separately
+                                 ;; in this example.
+                                 {:base-rulebase deserialized-rulebase})))
+
+{% endhighlight %}
+
+This example is the same as before, except the session rulebase and working memory are serialized separately.  This is commonly a very useful feature.  It is often the case that a "base" session is created that constructs a rulebase (Rete-based rule/query graph).  This base session is then used for mutually exclusive working memory datasets e.g. per-person domain data processed against the same set of rules.  For large sets of rules/queries, the rulebase can take non-trivial amounts of space and time to serialize and deserialize.  If the rulebase can be shared among different working memory states, Clara durability allows it to be stored separately.  When deserializing the working memory state to recreate a deserialized session, the deserialized rulebase just has to be provided via the `:base-rulebase` option passed to `d/deserialize-session-state` above.
+
```

Comment:
I wasn't referring to Cerner-style use-cases at all.  I was referring to the common goal of using the same rete graph for multiple, mutually exclusive datasets.

If I just say words like that with no concrete example, I think it may not make it clear what a case for that would be or what I was describing.  I like to have a concrete example to explain an abstract phrase like this.  If you have a better example?